<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuickSettingsWidget</class>
 <widget class="QWidget" name="QuickSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Performance</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QComboBox" name="perfPreset"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Performance Preset:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="systemSettingsGroup">
     <property name="title">
      <string>Graphics</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="3" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Bilinear Filtering</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Aspect Ratio</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Internal Resolution</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <layout class="QGridLayout" name="systemSettingsLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="vsync">
          <property name="text">
           <string>VSync</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="upscaleMultiplier"/>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="aspectRatio">
        <item>
         <property name="text">
          <string>Fit to Window / Fullscreen</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Auto Standard (4:3 Interlaced / 3:2 Progressive)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Standard (4:3)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Widescreen (16:9)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="bilinearFiltering">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string extracomment="Smooth: Refers to the texture clarity.">Bilinear (Smooth)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string extracomment="Sharp: Refers to the texture clarity.">Bilinear (Sharp)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_a">
     <property name="title">
      <string>Audio</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Volume:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSlider" name="volume">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="pageStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="volumeLabel">
          <property name="text">
           <string>100%</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="basicGroupBox">
     <property name="title">
      <string>Latency</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="0">
       <widget class="QGroupBox" name="horizontalGroupBox">
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Output Latency:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="outputLatency">
             <property name="minimum">
              <number>5</number>
             </property>
             <property name="maximum">
              <number>15</number>
             </property>
             <property name="value">
              <number>10</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
             <property name="tickInterval">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="outputLatencyLabel">
             <property name="text">
              <string>20 ms</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="optimalFramePacing">
        <property name="text">
         <string>Optimal Frame Pacing (Low Input Latency)</string>
        </property>
       </widget>
      </item>
      <item row="20" column="0" colspan="2">
       <layout class="QGridLayout" name="basicCheckboxGridLayout"/>
      </item>
      <item row="3" column="0">
       <widget class="QGroupBox" name="horizontalGroupBox">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Target Latency (Ouput Mixing Buffer):</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="targetLatency">
             <property name="minimum">
              <number>15</number>
             </property>
             <property name="maximum">
              <number>200</number>
             </property>
             <property name="value">
              <number>60</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
             <property name="tickInterval">
              <number>200</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="targetLatencyLabel">
             <property name="text">
              <string>60 ms</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
